get_openstack_info_open_port/setup.py

```diff
@@ -0,0 +1,12 @@
+import setuptools
+setuptools.setup(
+    scripts=['get_openstack_info_open_port/get_openstack_info_open_port.py'],
+    author="Antonio J. Delgado",
+    version='0.0.1',
+    name='get_openstack_info_open_port',
+    author_email="antonio.delgado@csc.fi",
+    url="",
+    description="This script will check that a given port in a given global (public) IP is open, and then search for an instance in Openstack with that IP, and find which rule and security group allow this incoming traffic.",
+    license="GPLv3",
+    keywords=["openstack", "security groups", "security", "neutron", "nova", "rule", "firewall"]
+)
```

Comment:
This is not super important but can be implemented if you like. 
You have add one requirements.txt file to install the dependencies.
You can add these dependencies in setup.py file too. Like below

...
    install_requires=[
        'docutils',
        'BazSpam ==1.1',
    ],

Reply:
This will not change the functionality of this package but will reduce the need of requirement.txt file. And then we dont have to manually install 'pip3 install -r requirements.txt'

Reply:
I have tested and found this working

```
(osclient) ➜  get_openstack_info_open_port git:(CCCP-3689) ✗ pip list installed | grep -i click
click                        8.0.1
click-config-file            0.6.0
(osclient) ➜  get_openstack_info_open_port git:(CCCP-3689) ✗ pip list installed | grep -i openstack
get-openstack-info-open-port 0.0.1
openstacksdk                 0.54.0
python-openstackclient       5.5.0
(osclient) ➜  get_openstack_info_open_port git:(CCCP-3689) ✗ pip list installed | grep -i neutronclient
python-neutronclient         7.5.0
(osclient) ➜  get_openstack_info_open_port git:(CCCP-3689) ✗
```

Reply:
Thanks! Added. Yes I forgot to put it there. I'll keep both, just in case. Since this is not expected to be really installed in the system but be part of a tool set. I have a skeleton of files that I use always to create a new project, and the setup is part of the skeleton.

Reply:
Yes I have found tool sets are added in setup.py file also.